plugins/ingest/build.gradle

```diff
@@ -23,8 +23,21 @@ esplugin {
 }
 
 dependencies {
+  compile 'org.jruby.joni:joni:2.1.6'
   testCompile 'org.elasticsearch:securemock:1.1'
 }
 
+task copyPatternsToResourcesDir(type: Copy) {
```

Comment:
I don't think the custom task isn't needed, if all resources are placed under `src/main/packaging/config`. 

@rjernst added logic that bundles custom files that is placed in `src/main/packaging` into the plugin zip file: https://github.com/elastic/elasticsearch/blob/master/buildSrc/src/main/groovy/org/elasticsearch/gradle/plugin/PluginBuildPlugin.groovy#L97